go.mod

```diff
@@ -13,12 +14,12 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.9.0
 	github.com/temporalio/features v0.0.0-20240806202554-bdfe567c9d89
-	go.temporal.io/api v1.36.0
-	go.temporal.io/sdk v1.28.1
+	go.temporal.io/api v1.38.0
+	go.temporal.io/sdk v1.28.2-0.20240829163125-e85a098eaee2
```

Comment:
Does Omes require a released version? If so, we can block this PR until the release is out.